src/tag/tag.style.ts

```diff
@@ -0,0 +1,134 @@
+                    backgroundColor = Color.Validation.Orange.Text;
+                    borderColor = Color.Validation.Orange.Text;
+                    break;
+                case "red":
+                    backgroundColor = Color.Validation.Red.Text;
+                    borderColor = Color.Validation.Red.Text;
+                    break;
+                case "blue":
+                    backgroundColor = Color.Validation.Blue.Text;
+                    borderColor = Color.Validation.Blue.Text;
+                    break;
+                default:
+                    backgroundColor = Color.Neutral[1];
+                    borderColor = Color.Neutral[1];
+                    break;
+            }
+        } else {
+            boxShadow = `0px 0px 4px 1px rgba(164, 164, 164, 0.70)`;
+            switch (props.$color) {
+                case "grey":
+                    backgroundColor = Color.Neutral[6];
+                    borderColor = Color.Neutral[4];
+                    color = Color.Neutral[3];
+                    break;
+                case "green":
+                    backgroundColor = Color.Validation.Green.Background;
+                    borderColor = Color.Validation.Green.Border;
+                    color = Color.Validation.Green.Text;
+                    break;
+                case "yellow":
+                    backgroundColor = Color.Validation.Orange.Background;
+                    borderColor = Color.Validation.Orange.Border;
+                    color = Color.Validation.Orange.Text;
+                    break;
+                case "red":
+                    backgroundColor = Color.Validation.Red.Background;
+                    borderColor = Color.Validation.Red.Border;
+                    color = Color.Validation.Red.Text;
+                    break;
+                case "blue":
+                    backgroundColor = Color.Validation.Blue.Background;
+                    borderColor = Color.Validation.Blue.Border;
+                    color = Color.Validation.Blue.Text;
+                    break;
+                default:
+                    backgroundColor = Color.Neutral[8];
+                    borderColor = Color.Neutral[5];
+                    color = Color.Neutral[1];
+                    boxShadow = `0px 0px 4px 1px rgba(164, 164, 164, 0.50)`;
+                    break;
+            }
+        }
+        return css`
+            background: ${backgroundColor};
+            border: 1px solid ${borderColor};
+            color: ${color};
+
+            @media (hover: hover) {
+                &:hover {
+                    box-shadow: ${props.$interactive ? boxShadow : "none"};
+                }
+            }
+        `;
+    }}
+
+    ${MediaQuery.MaxWidth.tablet} {
+        ${({ $interactive }) => {
+            if ($interactive) {
+                return css`
+                    ${TextStyleHelper.getTextStyle("H5", "semibold")};
+                    padding: 0.438rem 1rem;
```

Comment:
hmm this is an oddly specific padding. if you're trying to achieve an initial height of `2.5rem` you can try `padding: calc(0.5rem - 1px) 1rem;`